app/javascript/tests/patient/laboratory/FirstPositiveLaboratory.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Button } from 'react-bootstrap';
+import FirstPositiveLaboratory from '../../../components/patient/laboratory/FirstPositiveLaboratory';
+import { mockLaboratory1 } from '../../mocks/mockLaboratories';
+
+import LaboratoryModal from '../../../components/patient/laboratory/LaboratoryModal';
+
+function getWrapper(lab, onChange) {
+  return shallow(<FirstPositiveLaboratory lab={lab} onChange={onChange} />);
+}
+
+describe('FirstPositiveLaboratory', () => {
+  it('Properly renders a button to enter a lab result if lab is null', () => {
+    const wrapper = getWrapper(null);
```

Comment:
this test seems like its testing if the modal is opening.  I'd like to see that the button exists, button text is correct, and that the lab details and modal are hidden

Reply:
so this test is good because it tests the button opening the modal (not the edit button). I'd rename this and then add the test I described above

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/a5f7d0708f903826cb1f4dec5169309b37c77dbb#diff-30c5340a7ce13eb30d2abf50df7d2d5821ef5e3ff48852f2a949a137fbc55774R24